policies_test.go

```diff
@@ -110,6 +110,64 @@ func TestHostPolicy_TokenAware_SimpleStrategy(t *testing.T) {
 	iterCheck(t, iter, "2")
 }
 
+func TestHostPolicy_TokenAware_LWT_DisablesHostShuffling(t *testing.T) {
```

Comment:
In my view this tests the wrong thing, it should test the hosts you get when you fallback not if shuffling was called.

Reply:
Yes makes sense. Then we can remove the `shuffleReplicasFn`

Reply:
Problem is it's not deterministic. We could get the correct ordering by chance.

Reply:
This is now using a table driven `cmp.Diff` approach.